PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,27 +1,17 @@
 ## Description
-<!-- Please explain the changes you made here. -->
+
 
 ## Testing done
-<!-- Please explain local testing done, this can include
-automated testing, manual testing, cross browser testing -->
 
-## Testing Plan
-<!-- Please explain testing planned this can include
-automated testing, manual testing, cross browser testing -->
 
 ## Screenshots
-<!-- Please provide screenshots of UI changes if applicable -->
-
-## Acceptance Criteria (Definition of Done)
 
-#### Unique to this PR
-<!-- This would be a good place to include feature flag check item and info, specific dashboards and instrumentation check item and info -->
-- [ ] _Replace this line with individual tasks unique to your PR_
 
-#### Applies to all PRs
+## Acceptance criteria
+- [ ]
 
-- [ ] Appropriate logging
+## Definition of done
+- [ ] Events are logged appropriately
 - [ ] Documentation has been updated, if applicable
-- [ ] Provide link to originating GitHub issue, or connected to it via ZenHub
-- [ ] Does not contain any sensitive information (i.e. PII/credentials/internal URLs/etc., in logging, hardcoded, or in specs)
-- [ ] Provide which alerts would indicate a problem with this functionality (if applicable)
```

Comment:
How can we better express this requirement without removing it entirely? 

Where this ties into documenting server-side dependencies; it may be better addressed with a DowntimeNotification wrapper.

Where this ties to business metrics, what are the expectations for this new functionality? Can we track a specific metric or event to confirm that this is functioning (has this button been clicked, how often do we expect it to be clicked)? How often do we expect it to be executed, and what would indicate a problem?

Reply:
:thinking:

The `DowntimeNotification` wrapper is a good point. It's not exactly an alert and we won't know if it's being used (unless that's a thing we want it to do), but we could maybe add something in here about that. How does `Safety valves are in place in for when back end services are down` sound?

As far as metrics go, do we have guidelines on what events we should track? I think we keep track of all button clicks and things like that automagically, so this would be more manual "this thing happened" events. Would that be encompassed in the `Events are logged appropriately` item, or should it be separate?